src/style.css

```diff
@@ -0,0 +1,129 @@
+
+@font-face {
+  font-family: Avenir-Book;
+  src: url("./fonts/Avenir-Book.ttf");
+}
+
+@font-face {
+  font-family: Avenir-Heavy;
+  src: url("./fonts/Avenir-Heavy.ttf");
+}
+
+body {
+  font-family: Avenir-Book, sans-serif;
+  font-style: normal;
+  margin: 0;
+  padding: 0;
+}
+
+.input {
+  height: 68px;
+  width: 100%;
+  left: 0;
+  top: 0;
+  padding-left: 60px;
+  border-radius: 4px;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  font-family: sans-serif;
+  font-style: normal;
```

Comment:
it is default propertie

Reply:
when I delete this property , my font changes and not respond to the state change